inspirehep/config.py

```diff
@@ -1188,3 +1188,9 @@ def _(x):
     "stat.OT": "Other",
     "stat.TH": "Other"
 }
+
+# Import local config file if it is present.
+try:
+    from inspirehep.config_local import *
```

Comment:
Not a fan of this... we already have `inspirehep.cfg` in the `static` folder, why the duplicate? It will just add one step to debugging.